src/main/java/seedu/taassist/logic/commands/DeletecCommand.java

```diff
@@ -0,0 +1,49 @@
+package seedu.taassist.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import javafx.collections.ObservableList;
+import seedu.taassist.commons.core.Messages;
+import seedu.taassist.logic.commands.exceptions.CommandException;
+import seedu.taassist.model.Model;
+import seedu.taassist.model.moduleclass.ModuleClass;
+
+/**
+ * Deletes a moduleClass identified using it's className from TA-Assist.
+ */
+public class DeletecCommand extends Command {
+
+    public static final String COMMAND_WORD = "deletec";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
```

Comment:
I believe it is inconsistent with DeletecCommandParser logic? Shouldn't DeletecCommandParser read from preamble in this case? Also it should be noted that it also feels inconsistent with AddcCommand

Reply:
Yes, but I believe the `MESSAGE_USAGE` is contradictory with how it's actually parsed?

Reply:
I get what you mean now, will get it fixed!